/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:153: RuntimeWarning: invalid value encountered in double_scalars
  mean = mean_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:159: RuntimeWarning: invalid value encountered in double_scalars
  var = var_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:170: RuntimeWarning: invalid value encountered in double_scalars
  p_beta = quantile_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:153: RuntimeWarning: invalid value encountered in double_scalars
  mean = mean_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:159: RuntimeWarning: invalid value encountered in double_scalars
  var = var_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:170: RuntimeWarning: invalid value encountered in double_scalars
  p_beta = quantile_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:153: RuntimeWarning: invalid value encountered in double_scalars
  mean = mean_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:159: RuntimeWarning: invalid value encountered in double_scalars
  var = var_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:170: RuntimeWarning: invalid value encountered in double_scalars
  p_beta = quantile_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:153: RuntimeWarning: invalid value encountered in double_scalars
  mean = mean_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:159: RuntimeWarning: invalid value encountered in double_scalars
  var = var_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:170: RuntimeWarning: invalid value encountered in double_scalars
  p_beta = quantile_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:153: RuntimeWarning: invalid value encountered in double_scalars
  mean = mean_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:159: RuntimeWarning: invalid value encountered in double_scalars
  var = var_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:170: RuntimeWarning: invalid value encountered in double_scalars
  p_beta = quantile_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:153: RuntimeWarning: invalid value encountered in double_scalars
  mean = mean_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:159: RuntimeWarning: invalid value encountered in double_scalars
  var = var_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:170: RuntimeWarning: invalid value encountered in double_scalars
  p_beta = quantile_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:153: RuntimeWarning: invalid value encountered in double_scalars
  mean = mean_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:159: RuntimeWarning: invalid value encountered in double_scalars
  var = var_numerator / denominator
/storage/work/s/szk381/Py_mass_radius_working/PyCode/MLE_fit.py:170: RuntimeWarning: invalid value encountered in double_scalars
  p_beta = quantile_numerator / denominator
